composer.json

```diff
@@ -36,7 +36,9 @@
     "coreshop/resource-bundle": "^3.0",
     "coreshop/rule-bundle": "^3.0",
     "coreshop/pimcore-bundle": "^3.0",
-    "pimcore/pimcore": "^10.0"
+    "pimcore/pimcore": "^10.0",
+    "psr/http-client-implementation": "^1.0",
```

Comment:
This is the correct way to say we require a PSR-18 compatible client package. Pimcore already requires Symfony HTTP client directly.